internetarchive/cli/ia_upload.py

```diff
@@ -265,6 +265,15 @@ def main(argv, session):
                 if not identifier:
                     print('error: no identifier column on spreadsheet!')
                     sys.exit(1)
+                try:
+                    validate_ia_identifier(identifier)
+                except:
```

Comment:
PEP8 and https://realpython.com/the-most-diabolical-python-antipattern document why bare exceptions should be avoided.

Reply:
Thanks for pointing that out